compiler/src/compile.rs

```diff
@@ -1616,10 +1616,12 @@ impl<O: OutputStream> Compiler<O> {
             Comprehension { kind, generators } => {
                 self.compile_comprehension(kind, generators)?;
             }
-            Starred { value } => {
-                self.compile_expression(value)?;
-                self.emit(Instruction::Unpack);
```

Comment:
Would you mind removing this `Unpack` instruction now that we're not using it at all? You can just remove the enum variant in `bytecode/src/bytecode.rs` and the compiler should guide you in removing the everything else from there.

Reply:
Good call, will do.